llms/mlx_lm/utils.py

```diff
@@ -105,6 +107,7 @@ def generate(
     prompt: str,
     temp: float = 0.0,
     max_tokens: int = 100,
+    seed: int = 0,
```

Comment:
Also can you revert to set the seed outside this function (e.g. before you call `generate` in `generate.py`. I think I prefer that to give the caller more control over how to seed the RNG state.